docat/tests/test_upload_icon.py

```diff
@@ -0,0 +1,146 @@
+import io
+from unittest.mock import call, patch
+
+import docat.app as docat
+
+
+def test_successful_icon_upload(client_with_claimed_project):
+    upload_folder_response = client_with_claimed_project.post(
+        "/api/some-project/1.0.0", files={"file": ("index.html", io.BytesIO(b"<h1>Hello World</h1>"), "plain/text")}
+    )
+    assert upload_folder_response.status_code == 201
+
+    with patch("shutil.copyfileobj") as copyfileobj_mock:
+        with patch("os.remove") as remove_file_mock:
```

Comment:
Try not to nest `with patch` but rather use `with patch("") as patch1, patch("") as patch2`

Reply:
I searched for something like this but didn't find it at first :+1: